Makefile

```diff
@@ -8,6 +8,9 @@ SQLITE_EXEC ?= ./target/debug/limbo
 # Static library to use for SQLite C API compatibility tests.
 SQLITE_LIB ?= ./target/debug/liblimbo_sqlite3.a
 
+# Reference implementation to check against
+ACTUAL_SQLITE_EXEC ?= sqlite3
```

Comment:
If we are doing this, then we might as well have `SQLITE_EXEC` and `LIMBO_EXEC`.